packages/app/src/systems/Chains/eth/containers/ManageEthAssetsDialog.tsx

```diff
@@ -0,0 +1,176 @@
+import { cssObj } from '@fuel-ui/css';
+import {
+  Box,
+  CardList,
+  Dialog,
+  Icon,
+  Image,
+  Input,
+  Text,
+  Form,
+  IconButton,
+  Avatar,
+} from '@fuel-ui/react';
+import { NativeAssetId } from 'fuels';
+import { Controller, useWatch } from 'react-hook-form';
+
+import { EthAssetCard } from '../components';
+import type { SetAddressFormValues } from '../hooks';
+import { useManageEthAssets, useSetAddressForm } from '../hooks';
+
+import { store } from '~/store';
+import { shortAddress } from '~/systems/Core';
+
+export function ManageEthAssetsDialog() {
```

Comment:
I think this whole component `ManageEthAssetsDialog` is not needed.

we can use the same EthAssetsDialog, but creating an "editable" state for the screen.

- needs to add search input in `EthAssetsDialog`
- when it's on editable mode, it shows the remove buttons on list items and change the header.


on this way we centralize the behavior of the screen, avoiding duplication and different things between screens

also make layout more consistent / less movement